### DISCUSSION TITLE
Works out-of-the-box with new revision of E1

### DISCUSSION THREAD

**Discussion by mpfl** [thirtythreeforty/neolink#165] [General]
I took delivery of a Reolink E1 yesterday that appears to be a new hardware revision (it has a non-PoE Ethernet port).

I didn't do my homework properly and didn't realise that the E1 doesn't support RSTP (I use motionEye as an NVR). But I discovered neolink and had the container up-and-running in a couple of minutes.

Seems to be working flawlessly and I'm grateful for your hard work @thirtythreeforty.

**Comment by thirtythreeforty** [comment]
Awesome! Not just me, there's several others who pitched in :)

**Comment by stegl** [comment]
so you run it in docker?

**Reply by mpfl** [reply]
Yes, I do. I've just submitted my docker-compose configuration as a pull request (#169).
